Trying it up.

First notice: `npm test` fails.

```
verity:logbookr jeko$ npm test

> logbookr@0.1.0 test /Users/jeko/Documents/logbookr
> mocha mocha-tests



  LocalStorage
    converters
      .startsWith
        ✓ returns true if string starts with beginning
        ✓ returns false otherwise
      .shouldConvert()
        1) returns true on keys that are not prefixed with `config.dataPrefix`
        2) returns false otherwise
      .convert()
        returns an object
          ✓ has only requried keys
          ✓ all values are strings, null is allowed as item
          ✓ `key` is dataPrefix + key
        correctly converts…
          ✓ ChTr
          ✓ ChTr-{id}
          ✓ Procs
          3) Procs-{id}
          ✓ returns null as key/item for other types of keys

  FieldModel
    ✓ exports available field types
    ✓ has `name`, `type`, `description`
    ✓ `type` defaults to TEXT
    #validate()
      ✓ `type` must be one of FieldModel.types
      ✓ `name` must be non-empty string
      ✓ returns undefined on well-formed fields
    #defaultValue()
      ✓ empty array for PHOTOS
      ✓ empty string for other types

  ProcedureModel
    defaults() / initialize()
      ✓ defines meta info: `id`, `createdAt`, `requiredFields` and `fields`
      ✓ each required field has an according model attribute
      ✓ when no options.template is specified, attribute `fields` is an emtpy array
      ✓ when options.template is specified, copies over fields from options.template.get('fields') to `fields` attribute
      #clone()
        ✓ returns new instance with deep cloned attributes
      #diff()
        ✓ returns set of changes required to revert model to its previous state
        ✓ returns `false` when model is unchagned compared to its previous state
      #revert()
        ✓ returns model to its previous state (before latest `set()`)
        ✓ does nothing if model was not changed
      #safeSet()
        ✓ applies changes and returns true
        ✓ preserves model.validationError
        ✓ returns false and reverts changes if, when applied, resulted in invalid model
        ✓ throws Error if changes argument is incorrect
      #fieldInfo()
        ✓ returns field info by its name
        ✓ returns undefined if fields was not found

  tr()
    ✓ returns `en` translations
    ✓ returns `fr` translations
    ✓ warns about missing locales
    ✓ warns about missing keys


  36 passing (125ms)
  3 failing

  1) LocalStorage converters .shouldConvert() returns true on keys that are not prefixed with `config.dataPrefix`:
     TypeError: undefined is not a function
      at shouldConvert (src/local-storage/converters.js:21:21)
      at Context.<anonymous> (mocha-tests/local-storage.test.js:24:16)

  2) LocalStorage converters .shouldConvert() returns false otherwise:
     TypeError: undefined is not a function
      at shouldConvert (src/local-storage/converters.js:21:21)
      at Context.<anonymous> (mocha-tests/local-storage.test.js:29:16)

  3) LocalStorage converters .convert() correctly converts… Procs-{id}:

      Error: expected { id: '30b241d9-3294-1e9d-f193-8385a9e34afb',
  createdAt: 1445273520000,
  date: '19-10-2015 19:52',
  type: 'Discectomie simple',
  patient: 'deep tree',
  photos: [],
  diagnostic: '',
  supervision: 'Aidé',
  senior: '',
  stage: 'c',
  comment: '',
  fields:
   [ { name: 'diagnostic',
       description: 'Diagnostic',
       type: 'text' },
     { name: 'supervision',
       description: 'Supervision',
       type: 'choicetree' },
     { name: 'senior',
       description: 'Senior',
       type: 'choicetree' },
     { name: 'stage',
       description: 'Stage',
       type: 'choicetree' },
     { name: 'comment',
       description: 'Comment',
       type: 'textarea' } ],
  requiredFields:
   [ { name: 'date',
       description: 'Date Performed',
       type: 'date' },
     { name: 'type',
       description: 'Type of Procedure',
       type: 'choicetree' },
     { name: 'patient',
       description: 'Patient\'s Full Name',
       type: 'text' },
     { name: 'photos',
       description: 'Photos',
       type: 'photos' } ] } to sort of equal { id: '30b241d9-3294-1e9d-f193-8385a9e34afb',
  createdAt: 1445266320000,
  type: 'Discectomie simple',
  date: '19-10-2015 19:52',
  patient: 'deep tree',
  photos: [],
  requiredFields:
   [ { name: 'date',
       description: 'Date Performed',
       type: 'date' },
     { name: 'type',
       description: 'Type of Procedure',
       type: 'choicetree' },
     { name: 'patient',
       description: 'Patient\'s Full Name',
       type: 'text' },
     { name: 'photos',
       description: 'Photos',
       type: 'photos' } ],
  diagnostic: '',
  supervision: 'Aidé',
  senior: '',
  stage: 'c',
  comment: '',
  fields:
   [ { description: 'Diagnostic',
       name: 'diagnostic',
       type: 'text' },
     { description: 'Supervision',
       name: 'supervision',
       type: 'choicetree' },
     { description: 'Senior',
       name: 'senior',
       type: 'choicetree' },
     { description: 'Stage',
       name: 'stage',
       type: 'choicetree' },
     { description: 'Comment',
       name: 'comment',
       type: 'textarea' } ] }
      + expected - actual

       {
         "comment": ""
      -  "createdAt": 1445273520000
      +  "createdAt": 1445266320000
         "date": "19-10-2015 19:52"
         "diagnostic": ""
         "fields": [
           {

      at Assertion.assert (node_modules/expect.js/index.js:96:13)
      at Assertion.eql (node_modules/expect.js/index.js:230:10)
      at mocha-tests/local-storage.test.js:80:51
      at Array.forEach (native)
      at Context.<anonymous> (mocha-tests/local-storage.test.js:78:28)



npm ERR! Test failed.  See above for more details.
```

--- a/mocha-tests/legacy-data.js
+++ b/mocha-tests/legacy-data.js
@@ -1,3 +1,5 @@
+var moment = require('moment');
+
 module.exports = {
   ChTr: {
     key: 'ChTr',
@@ -22,7 +24,7 @@ module.exports = {
     item: '{"id":"30b241d9-3294-1e9d-f193-8385a9e34afb","type":"Discectomie simple","date":"19-10-2015 19:52","patient":"deep tree","diagnostic":"","supervision":"Aidé","senior":"","stage":"c","comment":""}',
     converted: {
       id: '30b241d9-3294-1e9d-f193-8385a9e34afb',
-      createdAt: 1445266320000,
+      createdAt: moment('19-10-2015 19:52', 'DD-MM-YYYY HH:mm').valueOf(),
       type: 'Discectomie simple',
       date: '19-10-2015 19:52',
       patient: 'deep tree',
@@ -89,7 +91,7 @@ module.exports = {
     item: '{"id":"4825a7f7-d04c-1cc8-506f-c265d367c549","type":"Arthrodèse antérieure","date":"12-10-2015 19:45","patient":"1","diagnostic":"diag","supervision":"Aidé","senior":"lalalala","stage":"hello","comment":"comment", "picture": "file://example/dir/pic.jpg"}',
     converted: {
       id: '4825a7f7-d04c-1cc8-506f-c265d367c549',
-      createdAt: 1444661100000,
+      createdAt: moment('12-10-2015 19:45', 'DD-MM-YYYY HH:mm').valueOf(),
       type: 'Arthrodèse antérieure',
       date: '12-10-2015 19:45',
       patient: '1',

--- a/src/local-storage/converters.js
+++ b/src/local-storage/converters.js
@@ -18,7 +18,7 @@
       startsWith: startsWith,
 
       shouldConvert: function (key) {
-        return !key.startsWith(config.dataPrefix);
+        return !startsWith(key, config.dataPrefix);
       },
 
       convert: function (key, item) {

